### PR TITLE
Register components manually

### PR DESCRIPTION
**Pull request by SaitTalhaNisanci**
Component Governance detector cannot detect git components currently, so
we should register them manually for SDL.

Example result: https://msdata.visualstudio.com/Database%20Systems/_componentGovernance/citus-ha?_a=components&typeId=4985361&alerts-view-option=active

We also have `pg` and `subcommands`, as far as I remember now we kinda own `subcommands` and for `pg` we seem to take only two files, it seemed not necessary to add whole postgres repo for this.

### CODE REVIEW

File: cgmanifest.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "Registrations": [ 
+        {
+            "Component": { 
+                "Type": "git", 
+                "git": {
+                    "RepositoryUrl": "https://github.com/rxi/log.c",
+                    "CommitHash": "f9ea34994bd58ed342d2245cd4110bb5c6790153"
+                }
+            },
+            "DevelopmentDependency": true
```
Comment (SaitTalhaNisanci) [inline cgmanifest.json:11]: This should probably be false

Comment (JelteF) [reply cgmanifest.json:11]: yes, I agree

File: cgmanifest.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            "Component": { 
+                "Type": "git", 
+                "git": {
+                    "RepositoryUrl": "https://github.com/postgres/postgres",
+                    "CommitHash": "9213462c539e6412fe0498a7f8e20b662e15c4ec"
+                }
+            },
+            
+            "license": "PostgreSQL",
+            "licenseDetail": [
```
Comment (SaitTalhaNisanci) [inline cgmanifest.json:33]: For some reason, NOTICE generation doesn't automatically add this definition. But we can manually append this to the NOTICE file. It is probably okay for it to be not automated as we will generate the notice file only once.

**Review by JelteF — CHANGES_REQUESTED** [review]
Missing this dependency: https://github.com/mattiasgustavsson/libs

Which we use for parsing .ini files.

**Review by JelteF — APPROVED** [review]

**Comment by DimCitus** [comment]
Thanks team!

**Pull request merged**